app/grandchallenge/reader_studies/utils.py

```diff
@@ -15,9 +14,8 @@ def migrate_reader_study_to_display_sets(rs, view_content):  # noqa: C901
             ds = DisplaySet.objects.create(reader_study=rs)
             images = []
             for key in item:
-                try:
-                    image = Image.objects.get(name=item[key])
-                except Image.DoesNotExist:
+                image = rs.images.filter(name=item[key]).first()
+                if image is None:
```

Comment:
Couldn't work out why it was there so removed it, test still passed.

Reply:
I put it there just in case there was a hanging list with images in it that are no longer part of the reader study, or just plain non-existent. Since the hanging list field is basically a free json field.

Reply:
That is checked in `hanging_list_valid`, so it shouldn't be possible for things that keep that validation. Either way I now made the function more conservative as we are changing data, so won't migrate a reader study if that occurs.